llmc/utils.h

```diff
@@ -15,6 +15,10 @@
 // implementation of dirent for Windows is in dev/unistd.h
 #ifndef _WIN32
 #include <dirent.h>
+#include <arpa/inet.h>
+#else
```

Comment:
You may be able to move the winsock2.h file into dev/unistd.h. I would try to remove the pragma (if possible) and use an explicit /link Ws2_32.lib in LDLIBS in the Windows section of the Makefile (but have not tested this).